CONTRIBUTING.md

```diff
@@ -651,3 +651,79 @@ Names of files and directories should be in lowercase, with dashes between words
+
+There are some things you can do to help speed up the process of your PR being merged though. In order to speed the process up, you need to know what needs to happen before a committer will actually hit the merge button. This section intends to give a little overview and insight of what happens after you create your PR.
+
+### The committer's perspective
+
+PRs have varying quality and even the best people make mistakes. It is the role of the committer team to assess whether any PR's changes are good changes or not. In order for any PR to be merged, at least one committer needs to be convinced of its quality enough to merge it.
+
+Committers typically assess three aspects of your PR:
+
+1. Whether the change's intention is necessary and desirable
+2. Whether the code quality of your changes is good
+3. Whether the artefacts produced by the code are good
+
+If you want your PR to get merged quickly and smoothly, it is in your best interest to help convince committers in these three aspects.
+
+### How to help committers assess your PR
+
+For the committer to judge your intention, it's best to explain why you've made your change. For trivial changes like version updates, you can omit this part of course because the intention is obvious but for anything more nuanced or even major version upgrades, it helps if you explain the background behind your change a bit. E.g. if you're adding a package, explain what it is and why it should be in Nixpkgs.
+This goes hand in hand with [Writing good commit messages](#writing-good-commit-messages).
+
+For the code quality assessment, you cannot do anything yourself as only the committer can do this and they already have your code to look at.
+In order to minimise the need for back and forth though, do take a look over your code changes yourself and try to put yourself into the shoes of someone who didn't just write that code. Would you immediately know what the code does by glancing at it? If not, reviewers will notice this and will ask you to clarify the code by refactoring it and/or adding a few explanations in code comments. Doing this preemptively can save you and the committer a lot of time.
+
+The code artefacts are the hardest for committers to assess because PRs touch all sorts of components: applications, libraries, NixOS modules, editor plugins and many many other things. Any individual committer can only really assess components that they themselves know how to use however and yet they must still be convinced somehow. There isn't a good generic solution to this but there are some ways easing the committer's job here:
+
+- Provide smoke tests that the committer can run without much research or setup.
```

Comment:
```suggestion
- Provide smoke tests that the committer can run without much research or setup.

```

This will make a paragraph under the bullet item instead of having a huge bullet item.

Reply:
I had placed a double space at the end of this to force a line break.

TIL nixpkgs explicitly disallows that:

https://github.com/NixOS/nixpkgs/blob/4e74b7332b0a3cd83c6b6f501a7dfa184a103c36/.editorconfig#L66-L72